tempest/api/workloadmgr/base.py

```diff
@@ -3967,3 +3971,10 @@ def delete_secret_order(self, order_uuid):
         resp = self.order_client.delete_order(order_uuid)
         LOG.debug(f"resp {resp}")
         return resp
+
+    '''
+    This method will add additional security groups to instance
+    '''
+    def add_security_group_to_instance(self, instance_id, sgid):
+        self.servers_client.add_security_group(instance_id, name=sgid)
```

Comment:
In case any exception occurs in this call, it needs to be handled gracefully in the same function itself.

Reply:
handled change in code.